src/components/customCharts/SimpleBarChart.tsx

```diff
@@ -1,21 +1,25 @@
-import React, { useState, useEffect, useRef, memo } from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import { Paper } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import RemoveIcon from '@material-ui/icons/Remove';
 import AddIcon from '@material-ui/icons/Add';
 import IconButton from '@material-ui/core/IconButton';
-import { getClickPosition } from './helpers';
+import {
+  getClickPosition,
+  drawLine,
+  setFontStyle,
+} from './helpers';
 
 interface Props {
   chartData: any[];
   recordSelectCallback: (arg: any) => any;
   selectedRecordKey: string;
-  width?: number,
-  height?: number,
-  spaceBetweenBars?: number,
-  barWidth?: number,
-  barColor?: string,
-  selectedBarColor?: string,
+  width?: number;
+  height?: number;
+  spaceBetweenBars?: number;
+  barWidth?: number;
+  barColor?: string;
+  selectedBarColor?: string;
 }
```

Comment:
Ja bym ten interfejs gdzieś do odzielnego pliku dodał

Reply:
Fakt że się powtarzają w innych customowych wykresach ale cała logika kontenera dla canvas się powtarza - wydzielę później całość do oddzielnego pliku ale jeszcze nie zdążyłem i chciałem zebrać więcej przykładowych wykresów żeby wiedzieć jak to najlepiej wydzielić.